src/global/install.rs

```diff
@@ -609,3 +614,127 @@ fn specs_match_local_environment<T: AsRef<RepoDataRecord>>(
     // the environment doesn't contain records that don't match the manifest
     remaining_prefix_records.is_empty()
 }
+
+#[cfg(test)]
+mod tests {
+    use indexmap::IndexMap;
+    use rattler_lock::LockFile;
+
+    use super::*;
+
+    #[test]
+    fn test_local_environment_matches_spec() {
+        let specs = IndexMap::from([(
+            PackageName::from_str("ripgrep").unwrap(),
+            MatchSpec::from_str("ripgrep=14.1.0", ParseStrictness::Strict).unwrap(),
+        )]);
+
+        let lock_file =
+            LockFile::from_str(include_str!("./test_data/lockfiles/ripgrep.lock")).unwrap();
+        let environment = lock_file.default_environment().unwrap();
+        let records = environment
+            .conda_repodata_records_for_platform(Platform::Linux64)
+            .unwrap()
+            .unwrap();
```

Comment:
This is duplicated a few times, maybe move it into a function?

Reply:
You could also check rstest fixtures.

Reply:
I wasn't aware that `rstest` supports fixtures. This is indeed much nicer!